File: airport-net/airport-net/Models/Ticket.cs
namespace airport_net.Models
{
    public class Ticket
    {
        public int Id { get; set; }
        public int NumberOfFlight { get; set; }
        public int NumberOfPlace { get; set; }
        public int Price { get; set; }
        public bool isBusy { get; set; }
        public int? FlightId { get; set; }
        public Flight? Flight { get; set; }
        public int? UserId { get; set; }
        public User? User { get; set; }

    }
}

File: airport-web/src/app/shared/components/main-layout/shared/interfaces/ticket.ts
import {Flight} from "./flight";
import {User} from "../../../admin-layout/shared/components/employee/shared/interfaces/User";

export interface Ticket {
  id?: number,
  numberOfFlight?: number,
  numberOfPlace?: number,
  price: number,
  isBusy?: boolean,
  flightId: number,
  flight?: Flight,
  user?: User
}
